lib/mongo/error/invalid_replacement_document.rb

```diff
@@ -19,23 +19,15 @@ module Mongo
   class Error
 
     # Exception raised if the object is not a valid replacement document.
-    #
-    # @since 2.0.0
     class InvalidReplacementDocument < Error
 
-      # The error message.
-      #
-      # @since 2.0.0
-      MESSAGE = 'Invalid replacement document provided'.freeze
```

Comment:
I am aware that this is technically in the public API but it shouldn't be so I don't have a problem removing it.

Reply:
Am I right that this exception was never raised before this PR? If yes, I think we can remove this constant.

Reply:
I personally would leave it with a `@deprecated` note, since it doesn't appear to be in the way.

Reply:
okay great